solidity/contracts/WALTHEA.sol

```diff
@@ -13,72 +13,41 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-pragma solidity >=0.4.22;
-
-/// This contract is a copy of the WETH9 contract found at https://github.com/gnosis/canonical-weth/blob/master/contracts/WETH9.sol
-/// with the only notable changes being a solidity compiler version relaxation and minimal syntactic changes to ensure
-/// the contract compiles and produces no warnings.
-contract WETH9 {
-    string public name     = "Wrapped Ether";
```

Comment:
name, symbol, and decimals are all implemented in openzeppelin's ERC20.sol. Name and Symbol are set by the constructor, decimals is a function hard-coded to return 18.